### PR TITLE
Create bookportuguese.md

### PR DESCRIPTION
**Pull request by romefeller**
Here is the post for your approval.

Alexandre.

### CODE REVIEW

**Comment by snoyberg** [comment]
Can you also update the `posts.yaml` and `authors.yaml` files?

**Comment by romefeller** [comment]
> Can you also update the `posts.yaml` and `authors.yaml` files?

Done.

File: blog/posts.yaml [hunk 1]
```diff
@@ -1,3 +1,7 @@
+- time: 2019-18-12T16:20:00Z
```
[the remark this answers is not in the document]
Comment (romefeller) [reply blog/posts.yaml]: Oops. Fixed this typo.

**Comment by snoyberg** [comment]
I've pushed a commit with a few tweaks. Let me know if you're happy with the changes, and then I'll merge and publish this.

**Comment by romefeller** [comment]
Great! You can merge and publish.

**Pull request merged**